### PR TITLE
Jp/variadics fix

### PR DESCRIPTION
**Pull request by JesusPinedaC**
This pull request strengths the use of variadics in DeeplayModules. Example:

``` python
class MyDeeplayModule(DeeplayModule):
    def __init__(self, *args, **kwargs):
        ...

class OtherDeeplayModule(DeeplayModule):
    def __init__(self, a, *args, **kwargs):
```

### CODE REVIEW

File: deeplay/external/external.py [hunk 1]
```diff
@@ -100,11 +101,24 @@ def build(self) -> nn.Module:
...
+
+            kwargs = self.kwargs.copy()
+            # hack for external
+            classtype = kwargs.pop("classtype")
+
+            self.__init__(*(classtype, self._args), **kwargs)
+            self._run_hooks("after_init")
+            self._is_constructing = False
+            self.__post_init__()
+
```
Comment (BenjaminMidtvedt) [inline deeplay/external/external.py]: Should probably not redefine the entire __construct__ method. Maybe we can abstract it into a separate `args, kwargs = self.get_init_arguments()` which can be different for DeeplayModule and External

File: test_variadics.ipynb
Comment (BenjaminMidtvedt) [inline test_variadics.ipynb:1]: delete

File: deeplay/external/external.py (hunk 1 above)
Comment (JesusPinedaC) [reply deeplay/external/external.py]: I agree. https://github.com/DeepTrackAI/deeplay/pull/60/commits/4141508bca7a59d1ac07075d9a7ea9bb44154448 corrects this!

**Review by BenjaminMidtvedt — COMMENTED** [review]
Looks great!

**Review by BenjaminMidtvedt — APPROVED** [review]

**Pull request merged**